### ISSUE TITLE
GVim or gVim - how to write it

### ISSUE THREAD

**Issue by scootergrisen** [vim/vim#3170]
Are there any correct way to spell gvim?

Should "GVim" be "gVim" in https://github.com/vim/vim/blob/master/runtime/gvim.desktop#L2

I see it spelled as "GVim" in 7 files.

runtime\doc\netbeans.txt

    highlighting is only defined for GVim.
    running in a color terminal and for GVim.

runtime\doc\todo.txt

    Patch for GVimExt to show an icon. (Dominik Riebeling, 2010 Nov 7)

runtime\doc\usr_09.txt

    Vim works in an ordinary terminal.  GVim can do the same things and a few

runtime\doc\vim-de.1

    Nur GTK\-GUI: Benutzt den GtkPlug\-Mechanismus, um GVim in einem anderen
    Systemweite Einstellungsdatei für GVim
    Persönliche Einstellungsdatei für GVim
    Systemweite Einstellungsdatei für das Menü von GVim

runtime\doc\vim-de.UTF-8.1

    Nur GTK\-GUI: Benutzt den GtkPlug\-Mechanismus, um GVim in einem anderen
    Systemweite Einstellungsdatei für GVim
    Persönliche Einstellungsdatei für GVim
    Systemweite Einstellungsdatei für das Menü von GVim

runtime\gvim.desktop

    Name=GVim

src\gui_gtk_x11.c

    * Obviously, this is a terrible hack making GVim depend on GTK's

Maybe check they are all spelled correct:

    ex

    view

    gvim
    gview
    evim
    eview

    rvim
    rview
    rgvim
    rgview

**Comment by scootergrisen** [comment]
runtime\doc\starting.txt says "EVim":

    EVim runs Vim as click-and-type editor.  This is very unlike the original Vi

And "Evim":

    In Evim these options are changed from their default value:

runtime\evim.vim says "Evim":

    " Vim script for Evim key bindings

**Comment by tonymec** [comment]
Hm, it seems that Bram writes "gvim" but "GvimExt" almost everywhere, but that the author of the CSApprox plugin consistently uses "GVim" — see quickfix list below.

This said, with Vim there usually isn't "only one way" of doing things, there often are several key sequences that give the same result, and as long as the text is clear I wouldn't be paranoid about which letter to write in uppercase — except in code examples, where case consistency is usually required (a variable named fooBar is not the same as Foobar or FooBar or foobar or FOOBAR or, of course, foo_bar). In code, and e.g. in examples of command-lines to invoke the program, the Vim executable name must be in all lowercase, i.e. vim, view, vimdiff, gvim, gview, etc.

Best regards,
Tony.

The following is what I got by `:helpgrep gvim\c` i.e. a caseless search on the string "gvim" through the installed help. (You might have slightly different result if you have a different choice of third-party plugins).
```
CSApprox.txt|1 col 26| *CSApprox.txt*  Bringing GVim colorschemes to the terminal!
CSApprox.txt|38 col 25| written to only support GVim, and don't work at all in terminal Vim.
CSApprox.txt|40 col 19| This plugin makes GVim-only colorschemes Just Work in terminal Vim, as long
CSApprox.txt|58 col 65| colorscheme.  To use this command, a user would generally start GVim, choose a
CSApprox.txt|71 col 14| and in GVim.  It will not work at all in a terminal with 16 or fewer
CSApprox.txt|77 col 56| NOTE: Although |:CSApproxSnapshot| can be used in both GVim and terminal Vim,
CSApprox.txt|81 col 69| terminal Vim to figure out what color would have been used in GVim.
CSApprox.txt|112 col 35| NOTE that creating snapshots with GVim will work better than making them with
CSApprox.txt|113 col 27| Vim, and (obviously) all "gvim" binaries are built with +gui.
CSApprox.txt|116 col 2| "gvim" binary, and not in their "vim" binary.  You can check if GUI support is
CSApprox.txt|125 col 9| have a "gvim" binary available, you can probably launch Vim with GUI support
CSApprox.txt|126 col 19| anyway by calling gvim with the |-v| flag in the shell: >
CSApprox.txt|127 col 5| gvim -v
CSApprox.txt|129 col 54| If the above works, you can remove the need to call "gvim -v" instead of "vim"
CSApprox.txt|130 col 53| all the time by creating a symbolic link from your "gvim" binary to "vim"
CSApprox.txt|132 col 24| sudo ln -s $(which gvim) $(which vim)
CSApprox.txt|134 col 18| If launching as "gvim -v" doesn"t work, and no package with GUI support is
CSApprox.txt|137 col 13| scheme from GVim to use its snapshot mode.  If this is inconvenient for you,
CSApprox.txt|139 col 53| conscious decision to build "vim" without +gui and "gvim" without terminal
CSApprox.txt|221 col 8| In gvim, setting a highlight group like "Visual" (the color of your visual
CSApprox.txt|291 col 71| actually affect the gui colors, in case the :gui is used to start gvim
CSApprox.txt|346 col 47| name might not match up perfectly between gvim (which uses the system's
CSApprox.txt|379 col 26| Vim, sticking instead to GVim.  Even now that nearly every popular terminal
CSApprox.txt|391 col 74| This is where CSApprox comes in.  It attempts to fill this void allowing GVim
CSApprox.txt|393 col 52| operation.  In the first mode, it attempts to make GVim colorschemes
CSApprox.txt|400 col 41| gives a very close approximation to the GVim colors without requiring any
CSApprox.txt|420 col 49| environment can't be configured to allow it (no GVim and no Vim with +gui, for
CSApprox.txt|470 col 23| snapshots made from GVim are immune to this problem.
CSApprox.txt|656 col 67| Fix a bug causing a syntax error when using GVim while
CSApprox.txt|668 col 41| when the user has gvim or a vim with gui support, but
mathmenu.txt|67 col 18| * via menu with gvim (see |mathmenu-menu|)
mathmenu.txt|68 col 42| These menus are active when one is in gvim's insert mode; then one
autocmd.txt|685 col 13| or when gvim regains input focus.
autocmd.txt|789 col 25| VimEnter when using gvim.  Can be used to
autocmd.txt|790 col 33| position the window from a .gvimrc file: >
debug.txt|73 col 70| You must obtain the debugger symbols (PDB) file for your executable: gvim.pdb
debug.txt|74 col 5| for gvim.exe, or vim.pdb for vim.exe. The PDB should be available from the
debug.txt|95 col 25| First launch vim.exe or gvim.exe and then launch Visual Studio.  (If you don't
debugger.txt|28 col 72| These features were added specifically for use in the Motif version of gvim.
debugger.txt|30 col 14| both vim and gvim.  Some of the other features could be used in the non-GUI
debugger.txt|71 col 23| work for both vim and gvim, whereas the debug sign is, in most cases, limited
debugger.txt|72 col 4| to gvim.  The one exception to this is Sun Microsystem's dtterm.  The dtterm
debugger.txt|129 col 11| compile a gvim which interfaces with VWS set the following flag, which sets
diff.txt|7 col 23| *diff* *vimdiff* *gvimdiff* *diff-mode*
diff.txt|34 col 19| You may also use "gvimdiff" or "vim -d -g".  The GUI is started then.
editing.txt|1574 col 19| Or, when starting gvim from a shell: >
editing.txt|1575 col 2| gvim file.log -c "set buftype=nofile"
eval.txt|4617 col 3| gvimrc file.  Use the |GUIEnter| autocommand to use this
eval.txt|6736 col 23| :echo remote_expr("gvim", "2+2")
eval.txt|6737 col 23| :echo remote_expr("gvim1", "b:current_syntax")
eval.txt|6793 col 22| :echo remote_send("gvim", ":DropAndReply ".file, "serverid").
eval.txt|6798 col 22| :echo remote_send("gvim", ":sleep 10 | echo ".
gui.txt|33 col 5| gvim [options] [files...]
gui.txt|38 col 16| *gui-init* *gvimrc* *.gvimrc* *_gvimrc* *$MYGVIMRC*
gui.txt|39 col 5| The gvimrc file is where GUI-specific startup commands should be placed.  It
gui.txt|40 col 72| is always sourced after the |vimrc| file.  If you have one then the $MYGVIMRC
gui.txt|51 col 55| the "no_buffers_menu" variable in your .vimrc (not .gvimrc!): >
gui.txt|58 col 15| - If the "-U {gvimrc}" command-line option has been used when starting Vim,
gui.txt|59 col 8| the {gvimrc} file will be read for initializations.  The following
gui.txt|60 col 39| initializations are skipped.  When {gvimrc} is "NONE" no file will be read
gui.txt|62 col 42| - For Unix and MS-Windows, if the system gvimrc exists, it is sourced.  The
gui.txt|63 col 39| name of this file is normally "$VIM/gvimrc".  You can check this with
gui.txt|66 col 12| - If the GVIMINIT environment variable exists and is not empty, it is
gui.txt|68 col 17| - If the user gvimrc file exists, it is sourced.  The name of this file is
gui.txt|69 col 22| normally "$HOME/.gvimrc".  You can check this with ":version".
gui.txt|71 col 14| - When a "_gvimrc" file is not found, ".gvimrc" is tried too.  And vice
gui.txt|73 col 52| The name of the first file found is stored in $MYGVIMRC, unless it was
gui.txt|75 col 70| - If the 'exrc' option is set (which is NOT the default) the file ./.gvimrc
gui.txt|77 col 3| gvimrc file.  If this file is not owned by you, some security restrictions
gui.txt|78 col 18| apply.  When ".gvimrc" is not found, "_gvimrc" is tried too.  For Macintosh
gui.txt|79 col 19| and DOS/Win32 "_gvimrc" is tried first.
gui.txt|92 col 17| You can use the gvimrc files to set up your own customized menus (see |:menu|)
gui.txt|97 col 19| Unix		    $HOME/.gvimrc or $HOME/.vim/gvimrc
gui.txt|98 col 19| OS/2		    $HOME/.gvimrc, $HOME/vimfiles/gvimrc
gui.txt|99 col 17| or $VIM/.gvimrc
gui.txt|100 col 29| MS-DOS and Win32    $HOME/_gvimrc, $HOME/vimfiles/gvimrc
gui.txt|101 col 17| or $VIM/_gvimrc
gui.txt|102 col 16| Amiga		    s:.gvimrc, home:.gvimrc, home:vimfiles:gvimrc
gui.txt|103 col 17| or $VIM/.gvimrc
gui.txt|165 col 4| in gvim. These may include an inability to enter spaces, or long delays
gui.txt|169 col 1| gvim from forking into the background by starting it with the |-f| argument.
gui.txt|241 col 62| then you can use the wheel to scroll the text up and down in gvim.  This works
gui.txt|258 col 24| 'mouse' option is the "gvimrc" file.
gui.txt|493 col 42| this line to your .vimrc file (NOT your .gvimrc file!): >
gui.txt|540 col 35| Accordingly, they are disabled if gvim is linked against GTK+ 3.4 or later.
gui.txt|936 col 38| mappings, or put these lines in your gvimrc; "<C-R>" is CTRL-R, "<CR>" is
gui.txt|1066 col 63| For the X11 GUI the external commands are executed inside the gvim window.
gui_w32.txt|31 col 45| return with a command prompt after starting gvim.  If not, you should use the
gui_w32.txt|33 col 8| start gvim [options] file ..
gui_w32.txt|49 col 10| vimrc or gvimrc file: >
gui_w32.txt|55 col 6| When gvim starts up normally, it creates its own top level window.  If you
gui_w32.txt|68 col 23| 3. Select the path to gvim for every file type that you want to use it for.
gui_w32.txt|72 col 2| gvim "%1"
gui_w32.txt|75 col 2| gvim "%L"
gui_w32.txt|83 col 2| gvim -c "simalt ~x" "%1"
gui_w32.txt|104 col 59| 4. Follow the shortcut wizard, using the full path to VIM/GVIM.
gui_w32.txt|111 col 36| You could replace notepad.exe with gvim.exe, but that has a few side effects.
gui_w32.txt|129 col 6| with gvim".  Choosing this menu option will invoke gvim with the file you have
gui_w32.txt|130 col 60| selected.  If you select multiple files, you will find two gvim-related menu
gui_w32.txt|132 col 21| "Edit with multiple gvims"  -- one gvim for each file in the selection
gui_w32.txt|133 col 19| "Edit with single gvim"     -- one gvim for all the files in the selection
gui_w32.txt|134 col 27| And if there already is a gvim running:
gui_w32.txt|135 col 21| "Edit with existing gvim"   -- edit the file with the running gvim
gui_w32.txt|138 col 34| registry under HKLM\Software\Vim\Gvim, named DisableEditWithExisting, and with
gui_w32.txt|153 col 25| {default}		    {path}\gvimext.dll
gui_w32.txt|155 col 52| HKEY_CLASSES_ROOT\*\shellex\ContextMenuHandlers\gvim
gui_w32.txt|160 col 36| HKEY_LOCAL_MACHINE\Software\Vim\Gvim
gui_w32.txt|161 col 21| path			    {path}\gvim.exe
gui_w32.txt|200 col 33| [HKEY_CLASSES_ROOT\Applications\gvim.exe]
gui_w32.txt|202 col 33| [HKEY_CLASSES_ROOT\Applications\gvim.exe\shell]
gui_w32.txt|204 col 33| [HKEY_CLASSES_ROOT\Applications\gvim.exe\shell\edit]
gui_w32.txt|206 col 33| [HKEY_CLASSES_ROOT\Applications\gvim.exe\shell\edit\command]
gui_w32.txt|207 col 20| @="c:\\vim\\vim62\\gvim.exe \"%1\""
gui_w32.txt|209 col 38| [HKEY_CLASSES_ROOT\.htm\OpenWithList\gvim.exe]
gui_w32.txt|211 col 35| [HKEY_CLASSES_ROOT\*\OpenWithList\gvim.exe]
gui_w32.txt|215 col 42| Change the "c:\\vim\\vim62" bit to where gvim.exe is actually located.
gui_w32.txt|439 col 9| And add gvim to the list of applications.  This problem only appears to happen
gui_w32.txt|445 col 1| Gvim can be build on MS-Windows with support for XPM files.  |+xpm_w32|
gui_x11.txt|28 col 5| gvim [options] [files...]
gui_x11.txt|31 col 32| So if you call the executable "gvim", or make "gvim" a link to the executable,
gui_x11.txt|33 col 14| added after "gvim", for example "gvim-5".
gui_x11.txt|38 col 3| :gvim [++opt] [+cmd] [-f|-b] [files...]			*:gv* *:gvim*
gui_x11.txt|45 col 6| When gvim was started from a shell this makes the shell accept further
gui_x11.txt|46 col 52| commands.  If you don't want this (e.g. when using gvim for a mail program
gui_x11.txt|47 col 16| that waits for gvim to exit), start gvim with "gvim -f", "vim -gf" or use
gui_x11.txt|51 col 13| When using "gvim -f" and then ":gui", Vim will run in the foreground.  The
gui_x11.txt|55 col 2| "gvim --nofork" does the same as "gvim -f".
gui_x11.txt|96 col 43| geometry		Initial geometry to use for gvim's window (default
gui_x11.txt|197 col 5| gvim -fg Yellow -bg Blue -reverse
gui_x11.txt|198 col 5| gvim -bg Yellow -fg Blue -reverse
gui_x11.txt|202 col 2| gvim -geometry 80x63+8+100
gui_x11.txt|323 col 52| -geom  or  -geometry    .geometry		size of the gvim window
hangulin.txt|56 col 27| If you use GTK version of gvim, you should set 'guifont' and 'guifontwide'.
hangulin.txt|61 col 39| If you use Motif or Athena version of gvim, you should set 'guifontset' in
hangulin.txt|64 col 8| $HOME/.gvimrc: >
hangulin.txt|80 col 60| And there should be no ':set guifont'.  If it exists, then gvim ignores
hangulin.txt|88 col 10| in your .gvimrc, then you should do for syntax >
if_ole.txt|131 col 2| gvim -register
if_ole.txt|133 col 4| If gvim with OLE support is run and notices that no Vim OLE server has been
if_ole.txt|138 col 73| registry is not writable.  If you run into this problem you need to run gvim
if_ole.txt|145 col 2| gvim -unregister
if_ole.txt|151 col 60| directory that is not in your normal path, so that typing "gvim" will start
if_ole.txt|157 col 2| gvim -silent -register
if_ole.txt|158 col 2| gvim -silent -unregister
if_ole.txt|177 col 31| Command   - c:\vim\vim63\gvim.exe
if_perl.txt|295 col 12| sure edit "gvim.exe" and search for "perl\d*.dll\c".
if_pyth.txt|740 col 39| (Python 3.6).  To know for sure edit "gvim.exe" and search for
if_ruby.txt|219 col 16| for sure edit "gvim.exe" and search for "ruby\d*.dll\c".
if_tcl.txt|537 col 7| edit "gvim.exe" and search for "tcl\d*.dll\c".
index.txt|1266 col 3| |:gvim|		:gv[im]		start the GUI
mbyte.txt|752 col 29| Then add a command to your |gvimrc| file to set 'guifont': >
mbyte.txt|999 col 54| CursorIM.  For example, add these lines to your |gvimrc|: >
message.txt|149 col 1| gvim.
message.txt|334 col 2| "gvim" and ":gui" don't work.
netbeans.txt|123 col 4| by gvim when it is run with one of the following GUIs: GTK, GNOME, Windows,
netbeans.txt|412 col 36| highlighting is only defined for GVim.
netbeans.txt|414 col 39| running in a color terminal and for GVim.
netbeans.txt|872 col 34| when the ":gui" command sources .gvimrc and this file loads a colorscheme
netbeans.txt|886 col 17| export SPRO_GVIM_DEBUG=netbeans.log
netbeans.txt|887 col 17| export SPRO_GVIM_DLEVEL=0xffffffff
netbeans.txt|927 col 20| NetBeans work with gvim (or xemacs :-).  See http://externaleditor.netbeans.org
netbeans.txt|989 col 5| the gvim command line is properly configured for your environment.
netbeans.txt|998 col 72| set up.  You can change the command name but that's about it.  If your gvim
netbeans.txt|999 col 65| can be found by your $PATH then the Vim Command can start with "gvim".  If
netbeans.txt|1000 col 16| you don't want gvim searched from your $PATH then hard code in the full
netbeans.txt|1001 col 49| Unix path name.  At this point you should get a gvim for any source file
netbeans.txt|1004 col 26| If some files come up in gvim and others (with different file suffixes) come
options.txt|144 col 4| $GVIMINIT or $EXINIT.
options.txt|892 col 46| "light".  When the value is not set in the .gvimrc, and Vim detects
options.txt|894 col 44| "dark".  But this happens only AFTER the .gvimrc file has been read
options.txt|897 col 35| putting a ":gui" command in the .gvimrc file, before where the value
options.txt|1722 col 56| to use the size for the GUI, put the command in your |gvimrc| file.
options.txt|1753 col 63| 'compatible' 'cp'	boolean	(default on, off when a |vimrc| or |gvimrc|
options.txt|1771 col 21| When a |vimrc| or |gvimrc| file is found while Vim is starting up,
options.txt|1774 col 26| that when a |vimrc| or |gvimrc| file exists, Vim will use the Vim
options.txt|1776 col 38| happen for the system-wide vimrc or gvimrc file, nor for a file given
options.txt|2948 col 44| Enables the reading of .vimrc, .exrc and .gvimrc in the current
options.txt|3836 col 17| e.g., in your |gvimrc| file.  When zero, the whole screen height will
options.txt|3905 col 12| can use "gvim -f" or ":gui -f" to start the GUI in the
options.txt|3908 col 30| happened already when the |gvimrc| file is read.
options.txt|3918 col 58| switching on syntax or filetype recognition (when the |gvimrc|
options.txt|3960 col 45| before starting the GUI.  Set it in your |gvimrc|.  Adding or
options.txt|4924 col 56| to use the size for the GUI, put the command in your |gvimrc| file.
options.txt|8775 col 38| might want to set it again in your |gvimrc|.
os_dos.txt|30 col 34| Your vimrc files ("_vimrc" and "_gvimrc") are normally located one directory
os_dos.txt|46 col 65| If you move your executable AND want to put your "_vimrc" and "_gvimrc" files
os_mac.txt|71 col 42| files are named "vimrc" or "_vimrc" and "gvimrc" or "_gvimrc".  These files
os_qnx.txt|49 col 50| To start the gui for vim, you need to run either gvim or vim -g, otherwise
os_vms.txt|131 col 5| gvimrc
os_vms.txt|179 col 36| You may want to create .vimrc and .gvimrc files in your home directory
os_vms.txt|183 col 32| (MENU.VIM) and files vimrc and gvimrc in the original $VIM directory.  It will
os_vms.txt|186 col 2| .gvimrc.  It should work without problems.
os_vms.txt|192 col 7| $VIM:gvimrc
os_vms.txt|198 col 13| sys$login:.gvimrc
os_vms.txt|251 col 11| vim -/u <gvimrc>  ! means: -U Use <gvimrc> instead of any .gvimrc
os_vms.txt|319 col 24| 8.8 Sourcing vimrc and gvimrc
os_vms.txt|454 col 2| .gvimrc: >
os_vms.txt|508 col 24| 8.8 Sourcing vimrc and gvimrc
os_vms.txt|510 col 37| If you want to use your .vimrc and .gvimrc from other platforms (e.g. Windows)
os_vms.txt|513 col 54| The symptom is that Vim is not sourcing your .vimrc/.gvimrc, even if you say:
os_vms.txt|542 col 12| When I use gvim this way and press CTRL-Y in the parent terminal, gvim exits.
os_vms.txt|548 col 54| The /INPUT=NLA0: separates the standard input of the gvim process from the
os_vms.txt|550 col 28| Without the -GEOMETRY, the gvim window size will be minimal and the menu
os_vms.txt|623 col 7| $ GVIMDIFF    :== 'GVIM' -D/R
os_vms.txt|630 col 8| 4. GVIM diff:  $ GVIMDIFF <FILE1> <FILE2>
os_vms.txt|680 col 8| gvimrc
os_vms.txt|681 col 8| gvim.exe (the renamed GUI or GTK built vim.exe)
os_vms.txt|688 col 22| $ gvi*m :== mcr VIM:GVIM.EXE
os_vms.txt|690 col 46| $ gv*im :== spawn/nowait/input=NLA0 mcr VIM:GVIM.EXE -g -GEOMETRY 80x40
os_vms.txt|912 col 27| - Updated files vimrc and gvimrc with VMS specific suggestions.
os_win32.txt|87 col 9| Vim and gvim used to use different logic to parse it (before 7.4.432), and the
os_win32.txt|88 col 68| logic was also depended on what it was compiled with.  Now Vim and gvim both
os_win32.txt|173 col 32| There was a special version of gvim that runs under Windows 3.1 and 3.11.
pi_netrw.txt|1079 col 24| <c-r>	Browse using a gvim server                           |netrw-ctrl-r|
pi_netrw.txt|1138 col 15| <leftmouse>	(gvim only) selects word under mouse as if a <cr>
pi_netrw.txt|1140 col 17| <middlemouse>	(gvim only) same as P selecting word under mouse;
pi_netrw.txt|1142 col 16| <rightmouse>	(gvim only) delete file/directory using word under
pi_netrw.txt|1144 col 17| <2-leftmouse>	(gvim only) when:
pi_netrw.txt|1151 col 17| <s-leftmouse>	(gvim only) like mf, will mark files.  Dragging
pi_netrw.txt|1242 col 21| When using the gui (gvim), one may select a file by pressing the <leftmouse>
pi_netrw.txt|1268 col 1| gvim, then the <s-cr> mapping may be used to squeeze (close) the
pi_netrw.txt|1341 col 18| BROWSING USING A GVIM SERVER			*netrw-ctrl-r* {{{2
pi_netrw.txt|1343 col 25| One may keep a browsing gvim separate from the gvim being used to edit.
pi_netrw.txt|1345 col 12| will use a gvim server (see |g:netrw_servername|).  Subsequent use of <cr>
pi_netrw.txt|1743 col 58| The <2-leftmouse> map (which is only available under gvim and
pi_netrw.txt|2116 col 9| * With gvim, in addition one may mark files with
pi_netrw.txt|2162 col 26| with <s-leftmouse> using gvim.  One may unmark all files by pressing
pi_netrw.txt|2450 col 36| * With mouse-enabled vim or with gvim, one may select a target by using
pi_netrw.txt|2457 col 22| When using menus and gvim, netrw provides a "Targets" entry which allows one
pi_netrw.txt|3502 col 33| In Preferences > Editors, set gvim as your editor:
pi_netrw.txt|3507 col 36| "c:\Program Files\Vim\vim70\gvim.exe" !.!
pi_netrw.txt|3516 col 37| to edit a file in a local copy of gvim.
pi_netrw.txt|3585 col 35| |CTRL-W_>|).  If you're using gvim, you can drag
pi_netrw.txt|3589 col 38| and |netrw-P|).  If you're using gvim, you can press
pi_netrw.txt|3705 col 22| P20. How may I have gvim start up showing a directory listing?
pi_netrw.txt|3710 col 29| \ (v:servername =~ 'GVIM\d*' || v:servername == "")
pi_netrw.txt|4082 col 39| cygwin+vim, but not for windows+gvim.
quickref.txt|1202 col 10| |-U|	-U {gvimrc}	    idem, for when starting the GUI
quotes.txt|83 col 62| This guide was written using the Windows 9.x distribution of gvim, which is
remote.txt|81 col 39| Edit "file.txt" in an already running GVIM server: >
remote.txt|82 col 5| gvim --remote file.txt
remote.txt|85 col 5| gvim --servername FOOBAR --remote file.txt
remote.txt|89 col 5| gvim --servername FILES --remote-silent file.txt
remote.txt|92 col 5| gvim --remote --servername FOOBAR file.txt
remote.txt|104 col 74| By default Vim will try to register the name under which it was invoked (gvim,
remote.txt|105 col 2| egvim ...).  This can be overridden with the --servername argument.  If the
remote.txt|107 col 20| encountered, i.e. "gvim1" for the second invocation of gvim on a particular
remote.txt|109 col 64| |v:servername|.  The case of the server name is ignored, thus "gvim" and
remote.txt|110 col 2| "GVIM" are considered equal.
remote.txt|120 col 55| itself.  This way it is not necessary to know whether gvim is already started
remote.txt|179 col 22| When an X11 GUI Vim (gvim) is started, it will try to register a send-server
remote.txt|203 col 12| When using gvim, the --remote-wait only works properly this way: >
remote.txt|205 col 11| start /w gvim --remote-wait file.txt
sign.txt|26 col 50| sign as an image is most likely only feasible in gvim (although Sun
starting.txt|99 col 1| gvim	vim -g	    Start the GUI (see |gui|).			    *gvim*
starting.txt|104 col 2| rgvim	vim -gZ	    Like "gvim", but in restricted mode.	  *rgvim*
starting.txt|109 col 1| gvimdiff vim -gd    Start in diff mode |diff-mode|
starting.txt|112 col 2| "gvim-5" to start the GUI.  You must have an executable by that name then, of
starting.txt|120 col 8| alias gvim   vim -g
starting.txt|265 col 33| executable is called "ex" or "gvim".  For gvim the GUI is not
starting.txt|386 col 8| in a gvimrc file.  See |startup|.
starting.txt|451 col 3| gvim is started by a program that will wait for the edit
starting.txt|452 col 60| session to finish (e.g., mail or readnews).  If you want gvim
starting.txt|453 col 55| never to fork, include 'f' in 'guioptions' in your |gvimrc|.
starting.txt|463 col 44| running Vim with an installed vim.bat or gvim.bat file it
starting.txt|482 col 35| skipped, including reading the |gvimrc| file when the GUI
starting.txt|499 col 5| -U {gvimrc}	The file {gvimrc} is read for initializations when the GUI
starting.txt|500 col 58| starts.  Other GUI initializations are skipped.  When {gvimrc}
starting.txt|519 col 9| - no |gvimrc| script is loaded
starting.txt|609 col 28| GTK+ GUI Vim only.  Make gvim try to use GtkPlug mechanism, so
starting.txt|614 col 29| Win32 GUI Vim only.  Make gvim try to use the window {id} as a
starting.txt|619 col 28| GTK+ GUI Vim only.  Make gvim echo the Window ID on stdout,
starting.txt|620 col 28| which can be used to run gvim in a kpart widget.  The format
starting.txt|926 col 22| Only when starting "gvim", the GUI initializations will be done.  See
starting.txt|954 col 20| The $MYVIMRC or $MYGVIMRC file will be set to the first found vimrc and/or
starting.txt|955 col 1| gvimrc file.
starting.txt|1021 col 5| - a gvimrc file was found,
starting.txt|1131 col 12| When using gvim, it detects that it might have been started from the desktop,
starting.txt|1138 col 9| :read !gvim --help
starting.txt|1140 col 41| This still won't work for systems where gvim does not use stdout at all
starting.txt|1225 col 52| In the GUI, suspending is implemented as iconising gvim.  In Windows 95/NT,
starting.txt|1226 col 1| gvim is minimized.
syntax.txt|17 col 14| GUI version, gvim.
syntax.txt|86 col 34| NOTE: When using reverse video ("gvim -fg white -bg black"), the default value
syntax.txt|88 col 14| reading the |gvimrc|.  This will cause the wrong default highlighting to be
syntax.txt|90 col 50| highlighting, include the ":gui" command in the |gvimrc|: >
syntax.txt|95 col 28| NOTE: Using ":gui" in the |gvimrc| means that "gvim -f" won't start in the
syntax.txt|413 col 24| for f in *.[ch]; do gvim -f +"syn on" +"run! syntax/2html.vim" +"wq" +"q" $f; done
syntax.txt|1435 col 1| gvim display.  Here, statements are colored LightYellow instead of Yellow, and
todo.txt|307 col 2| "gvim --remote" from a directory with non-word characters changes the current
todo.txt|537 col 43| MS-Windows: Opening same file in a second gvim hangs. (Sven Bruggemann, 2017
todo.txt|553 col 5| Use gvimext.dll from the nightly build? (Issue #249)
todo.txt|1080 col 1| Gvim: when both Tab and CTRL-I are mapped, use CTRL-I not for Tab.
todo.txt|1864 col 11| Patch for GVimExt to show an icon. (Dominik Riebeling, 2010 Nov 7)
todo.txt|1958 col 13| For running gvim on a USB stick: avoid the OLE registration.  Use a command
todo.txt|2010 col 25| When starting Vim with "gvim -f -u non_existent_file > foo.txt" there are a
todo.txt|2173 col 24| Wildmenu not deleted: "gvim -u NONE", ":set nocp wildmenu cmdheight=3
todo.txt|2249 col 15| Win32: Using "gvim --remote-tab-silent elŝuti.txt" doesn't work, the
todo.txt|2258 col 20| Only happens with "gvim -geometry "160x26+4+27" -u NONE -U NONE prop.c".
todo.txt|2441 col 73| Netbeans problem.  Use "nc -l 127.0.0.1 55555" for the server, then run gvim
todo.txt|2442 col 7| with "gvim -nb:localhost:55555:foo".  From nc do: '1:editFile!0 "foo"'.  Then
todo.txt|2447 col 1| Gvimext: patch to add "Edit with single Vim &tabbed" menu entry.
todo.txt|2483 col 15| Win32: using "gvim --remote-tab-silent fname" sometimes gives an empty screen
todo.txt|2523 col 1| Gvim: dialog for closing Vim should check if Vim is busy writing a file.  Then
todo.txt|2565 col 1| gvim d:\path\path\(FILE).xml should not remove the \ before the (.
todo.txt|2605 col 12| Mac: Using gvim: netrw window disappears. (Nick Lo, 2006 Jun 21)
todo.txt|2622 col 29| Michael Dietrich: maximized gvim sometimes displays output of external command
todo.txt|2756 col 4| In gvim the backspace key produces a backspace character, but on Linux the
todo.txt|2763 col 10| Create a gvimtutor.1 file and change Makefiles to install it.
todo.txt|2887 col 23| raisewin()		raise gvim window (see HierAssist patch for
todo.txt|2927 col 19| 7   Unix Vim (not gvim): Typing CTRL-C in Ex mode should finish the line
todo.txt|2987 col 45| 9   Can't paste a Visual selection from GTK-gvim to vim in xterm or Motif gvim
todo.txt|2988 col 60| when it is longer than 4000 characters.  Works OK from gvim to gvim and
todo.txt|2990 col 33| It starts working after GTK gvim loses the selection and gains it again.
todo.txt|3014 col 33| 8   Win32: When clicking on the gvim title bar, which gives it focus, produces
todo.txt|3015 col 74| a file-changed dialog, after clicking on a button in that dialog the gvim
todo.txt|3108 col 19| 5   When starting gvim with all the default colors and then typing
todo.txt|3170 col 50| lines below the text and bottom scrollbar.  "gvim -geometry 80x78+0+0".
todo.txt|3181 col 50| 7   GUI on Solaris 2.5.1, using /usr/dt/..: When gvim starts, cursor is
todo.txt|3503 col 50| 8   When "cm" termcap entry is missing, starting gvim shouldn't complain about
todo.txt|3558 col 14| -   In zsh, "gvim&" changes the terminal settings.  This is a zsh problem.
todo.txt|3595 col 36| -   Solaris 2.6 with GTK and Perl: gvim crashes when started.  Problem with X
todo.txt|3609 col 58| -   GTK: When pasting a selection from Vim to xclipboard gvim crashes with a
todo.txt|3715 col 66| 7   When Vim detects a file is being edited elsewhere and it's a gvim session
todo.txt|3716 col 74| of the same user it should offer a "Raise" button, so that the other gvim
todo.txt|4054 col 23| loaded (from the .gvimrc).	After changing 'background', load it again?
todo.txt|4518 col 5| 7   GvimExt: when there are several existing Vims, move the list to a submenu.
todo.txt|4567 col 74| 8   Make it possible to ignore a mouse click when it's used to give Vim (gvim)
todo.txt|4579 col 39| 8   Should take font from xterm where gvim was started (if no other default).
todo.txt|4591 col 23| Useful when using gvim to run a script (e.g. 2html.vim).
todo.txt|4600 col 9| 8   ":mkgvimrc" command, that includes menus.
usr_01.txt|128 col 59| On Unix, if you prefer using the GUI version of Vim, use "gvimtutor" or
usr_02.txt|30 col 2| gvim file.txt
usr_02.txt|57 col 5| The gvim command causes the editor to create a new window for editing.  If you
usr_02.txt|66 col 20| versions, but with gvim you have extra features, like a menu bar.  More about
usr_04.txt|363 col 42| If you are using the GUI version of Vim (gvim), you can find the "Copy" item
usr_06.txt|48 col 26| enable" command in your |gvimrc| file.
usr_09.txt|8 col 37| Vim works in an ordinary terminal.  GVim can do the same things and a few
usr_09.txt|24 col 52| You might have an icon on your desktop that starts gvim.  Otherwise, one of
usr_09.txt|27 col 2| gvim file.txt
usr_09.txt|187 col 66| switch to Visual mode and highlight the text.  Now start another gvim, without
usr_09.txt|192 col 47| selected.  After doing the paste in the other gvim, now select some characters
usr_09.txt|194 col 14| in the other gvim window are displayed differently.  This means that it no
usr_09.txt|207 col 5| the gvims you have running.  Then use the Edit/Copy menu entry.  Now the text
usr_09.txt|210 col 25| Now select the other gvim, position the cursor somewhere and use the
usr_09.txt|218 col 9| Use one gvim with a text file and perform these actions:
usr_21.txt|480 col 23| using something like "gvim:" will not work.
usr_31.txt|198 col 40| You can start the X-Windows version of gvim with an argument to specify the
usr_31.txt|201 col 2| gvim -geometry {width}x{height}+{x_offset}+{y_offset}
usr_31.txt|206 col 2| gvim -geometry 80x25+100+300
usr_31.txt|211 col 13| You can use gvim to edit an e-mail message.  In your e-mail program you must
usr_31.txt|212 col 8| select gvim to be the editor for messages.  When you try that, you will
usr_31.txt|214 col 7| while gvim is still running!
usr_31.txt|215 col 25| What happens is that gvim disconnects from the shell it was started in.
usr_31.txt|216 col 29| That is fine when you start gvim in a terminal, so that you can do other work
usr_31.txt|217 col 57| in that terminal.  But when you really want to wait for gvim to finish, you
usr_31.txt|220 col 2| gvim -f file.txt
usr_31.txt|239 col 5| THE GVIM STARTUP FILE
usr_31.txt|241 col 6| When gvim starts, it reads the gvimrc file.  That's similar to the vimrc file
usr_31.txt|242 col 30| used when starting Vim.  The gvimrc file can be used for settings and commands
usr_31.txt|250 col 8| The gvimrc file is searched for in the same locations as the vimrc file.
usr_31.txt|251 col 26| Normally its name is "~/.gvimrc" for Unix and "$VIM/_gvimrc" for MS-Windows.
usr_31.txt|252 col 8| The $MYGVIMRC environment variable is set to it, thus you can use this command
usr_31.txt|255 col 11| :edit $MYGVIMRC
usr_31.txt|257 col 56| If for some reason you don't want to use the normal gvimrc file, you can
usr_31.txt|260 col 2| gvim -U thisrc ...
usr_31.txt|262 col 22| That allows starting gvim for different kinds of editing.  You could set
usr_31.txt|264 col 33| To completely skip reading a gvimrc file: >
usr_31.txt|266 col 2| gvim -U NONE ...
usr_90.txt|236 col 2| gvim61.exe		The self-installing archive.
usr_90.txt|244 col 2| gvim61.zip		The normal MS-Windows GUI version.
usr_90.txt|245 col 2| gvim61ole.zip		The MS-Windows GUI version with OLE support.
usr_90.txt|265 col 13| unzip path\gvim61.zip
usr_90.txt|368 col 1| gvim.  This avoids the need for a correct terminal setup.
usr_90.txt|441 col 20| rm /usr/local/bin/gvim
usr_90.txt|442 col 20| rm /usr/local/bin/gvim
usr_90.txt|443 col 20| rm /usr/local/bin/gvimdiff
usr_90.txt|445 col 21| rm /usr/local/bin/rgvim
usr_90.txt|458 col 25| rm /usr/local/man/man1/gvim.1
usr_90.txt|459 col 25| rm /usr/local/man/man1/gvimdiff.1
usr_90.txt|461 col 26| rm /usr/local/man/man1/rgvim.1
version5.txt|208 col 3| a gvimrc file.
version5.txt|301 col 28| includes the global vimrc, gvimrc, and menu.vim files and all on-line help
version5.txt|414 col 44| It is possible to produce a DLL version of gvim with Borland C++ (Aaron).
version5.txt|607 col 15| |-U|		Set the gvimrc file to be used.  Like "-u" for the vimrc.
version5.txt|758 col 45| - "vim -u NONE" now also stops loading the .gvimrc and other GUI inits.  |-u|
version5.txt|759 col 27| Use "-U" to use another gvimrc file.  |-U|
version5.txt|781 col 43| Ignore characters after "ex", "view" and "gvim" when checking startup mode.
version5.txt|782 col 20| Allows the use of "gvim5" et. al.  |gvim| "gview" starts the GUI in readonly
version5.txt|1261 col 30| When $DISPLAY is not set and gvim starts vim, title was restored to "Thanks
version5.txt|1263 col 34| When $DISPLAY not set, starting "gvim" (dropping back to vim) and then
version5.txt|1520 col 28| Crash with ":unmenu *" in .gvimrc for Athena.
version5.txt|1591 col 2| .gvimrc. (Acevedo)
version5.txt|1839 col 7| Win32 gvim now only sets the console size for external commands to 25x80
version5.txt|1896 col 21| When starting with "gvim -bg black -fg white", the value of 'background' is
version5.txt|1897 col 29| only set after reading the .gvimrc file.  This causes a ":syntax on" to use
version5.txt|1899 col 33| colors.  Previously ":gui" in a gvimrc crashed Vim.
version5.txt|1908 col 42| colors that were set by the user in the .gvimrc file.  Now it only changes
version5.txt|1946 col 11| X11 GUI: "gvim -rv" reversed the colors twice on Sun.  Now Vim checks if the
version5.txt|2350 col 38| Win32 GUI: When directory dropped on Gvim, cd there and edit new buffer.
version5.txt|2543 col 22| 2html.vim outside of gvim (using color names instead of #RRGGBB).
version5.txt|2709 col 34| When using a ":set ls=1" in the .gvimrc file, would get a status line for a
version5.txt|3055 col 24| 5.2.3: Win32 GUI: When gvim was installed in "Program files", or another path
version5.txt|3575 col 48| Renamed vimrc_example to vimrc_example.vim and gvimrc_example to
version5.txt|3576 col 1| gvimrc_example.vim.  Makes them being recognized as vim scripts.
version5.txt|3593 col 54| would not be found.  Now use $HOME only for _vimrc, _gvimrc, etc., not to find
version5.txt|3631 col 36| To be used with the OLE version of gvim under MS-Windows. (Schaller)
version5.txt|3856 col 4| - "gvim -v" doesn't start the GUI (if console support is present).
version5.txt|3858 col 23| - When starting with "gvim -f" and using ":gui" in the .gvimrc file, Vim
version5.txt|3860 col 10| to run gvim in the background anyway.
version5.txt|4323 col 28| GUI: When checking for a ".gvimrc" file in the current directory, didn't check
version5.txt|4324 col 9| for a "_gvimrc" file too.
version5.txt|4511 col 22| - Dropping a file on gvim didn't activate the window.
version5.txt|4608 col 58| "make uninstall" didn't delete the vimrc_example.vim and gvimrc_example.vim
version5.txt|4670 col 1| gvimrc when it will not.  Also do normal terminal initializations if the GUI
version5.txt|4804 col 52| Win32 GUI: Dropping an encrypted file on a running gvim didn't work right.  Vim
version5.txt|4872 col 71| Problem:    Win32 GUI: When reading text from stdin, e.g., "cat foo | gvim -",
version5.txt|5136 col 24| - The system vimrc and gvimrc files were called ".vimrc" and ".gvimrc".
version5.txt|5247 col 6| gvim.exe when it wasn't moved and it's not in $PATH.
version5.txt|5370 col 43| Problem:    GTK GUI: Using ":gui" in the .gvimrc file caused an error.  Only
version5.txt|5395 col 49| Problem:    GTK GUI: When $DISPLAY is invalid, "gvim -f" just exits.  It
version5.txt|5441 col 43| Problem:    GUI: When started from inside gvim with ":!gvim", Vim would not
version5.txt|5442 col 17| start.  ":!gvim -f" works fine.
version5.txt|5461 col 32| Problem:    GTK: When starting gvim, would send escape sequences to the
version5.txt|5690 col 63| Problem:    Win32 GUI: When dropping a directory on a running gvim it crashes.
version5.txt|5888 col 12| Win32 GUI: gvimext.dll, for the context menu "Edit with Vim" entry.  Avoids
version5.txt|6123 col 21| "light" in the gvimrc, the highlighting isn't set.  (Tsjokwing)
version5.txt|6124 col 62| Solution:   Set the highlighting when 'bg' is changed in the gvimrc, even
version5.txt|6136 col 56| Problem:    Win32 GUI: When a file is dropped on Win32 gvim while at the ":"
version5.txt|6140 col 50| Solution:   When dropping a file or directory on gvim while at the ":" prompt,
version5.txt|6159 col 46| Problem:    Win32 GUI: When exiting Windows, gvim will leave swap files behind
version5.txt|6182 col 14| Problem:    "gvim -rv" caused a crash.  Using 't_Co' before it's set.
version5.txt|6342 col 52| Problem:    GTK GUI: When $term is invalid, using "gvim" gives an error
version5.txt|6584 col 48| Problem:    GUI GTK: Could not set display for gvim.
version5.txt|6631 col 43| Files:	    src/dosinst.c, src/uninstal.c, gvimext/*, runtime/doc/gui_w32.txt
version5.txt|6812 col 12| Files:	    GvimExt/gvimext.cpp
version5.txt|6979 col 59| Problem:    GTK GUI: Copy/paste text doesn't work between gvim and Eterm.
version5.txt|7046 col 20| Files:	    runtime/gvimrc_example.vim, runtime/vimrc_example.vim,
version5.txt|7273 col 69| Problem:    GUI: When interrupting an external program with CTRL-C, gvim might
version5.txt|7380 col 30| Problem:    Win32 GUI: "ls | gvim -" will show a message box about reading
version5.txt|7618 col 8| Win32: Gvimext could not edit more than a few files at once, the length of the
version6.txt|781 col 8| - Made gvimext.dll use translated messages, if possible. (Yasuhiro Matsumoto)
version6.txt|975 col 29| distribution.  Can now use "gvim --remote" and "gvim --remote-send", which is
version6.txt|1490 col 70| Also made it possible to delete only some of these.  Also unregister gvim for
version6.txt|1590 col 60| the menu 70% faster.  This can halve the startup time of gvim.
version6.txt|1758 col 7| the gvim window. (Brent Verner)
version6.txt|1955 col 31| - When dropping a shortcut on gvim (.lnk file) edit the target, not the
version6.txt|2174 col 21| MS-Windows: The big gvim.exe, which includes OLE, now also includes
version6.txt|2186 col 48| Win32: Added Makefile for Borland 5 to compile gvimext.dll. (Yasuhiro
version6.txt|2192 col 37| When checking the command name for "gvim", "ex", etc. ignore case.  Required
version6.txt|2441 col 25| GTK GUI: When starting "gvim" under some conditions there would be an X error.
version6.txt|2461 col 63| Motif GUI: When using the right mouse button, for some people gvim froze for
version6.txt|2544 col 64| The DOS install program: On Win2000 the check for a vim.exe or gvim.exe in
version6.txt|2905 col 61| When there was an error in a .vimrc file the terminal where gvim was started
version6.txt|2935 col 16| 'title' in the gvimrc and starting the GUI with ":gui".  Closed the connection
version6.txt|2970 col 8| Win32: Gvimext could not edit more than a few files at once, the length of the
version6.txt|3032 col 17| Win32: When the gvim window was maximized and selecting another font, the
version6.txt|3041 col 39| GTK: When adding a toolbar item while gvim is already running, it wasn't
version6.txt|3162 col 48| GTK: When setting 'columns' while starting up "gvim", would set the width of
version6.txt|3305 col 32| When a command in a .vimrc or .gvimrc causes an ATTENTION prompt, and Vim was
version6.txt|4088 col 17| Files:	    nsis/gvim.nsi, nsis/README.txt, src/dosinst.c
version6.txt|4804 col 41| Solution:   Exit when using "evim" and "gvim -y" when the GUI can't be
version6.txt|5033 col 14| Problem:    "gvim -f" still forks when using the batch script to start Vim.
version6.txt|5336 col 32| Problem:    GTK GUI: Can't use gvim in a kpart widget.
version6.txt|5812 col 28| Problem:    When compiling gvimext.cpp with MSVC 4.2 get a number of warnings.
version6.txt|5814 col 12| Files:	    GvimExt/gvimext.cpp
version6.txt|5907 col 33| Problem:    Win32: When closing gvim while it is minimized  and has a changed
version6.txt|5997 col 11| Moved the GvimEXt, OleVim and VisVim directories into the "src" directory.
version6.txt|6000 col 64| The VisVim.dll file is installed in the top directory, next to gvimext.dll,
version6.txt|6008 col 64| Changed the Win32 Makefiles to become more uniform and compile gvimext.dll.
version6.txt|6147 col 9| Compile gvimext.dll with MingW. (Rene de Zwart)
version6.txt|6183 col 24| Dosinst.exe also finds gvimext.dll in the "GvimExt" directory.  Useful when
version6.txt|6257 col 13| Win32: When gvim.exe is started from a shortcut with the window style property
version6.txt|7428 col 12| Files:	    GvimExt/gvimext.cpp, GvimExt/gvimext.h
version6.txt|8258 col 14| Problem:    "gvim --remote file" doesn't prompt for an encryption key.
version6.txt|8907 col 38| Problem:    MS-Windows: Cannot build GvimExt with MingW or Cygwin.
version6.txt|8910 col 12| Files:	    GvimExt/Make_cyg.mak, GvimExt/Make_ming.mak, GvimExt/Makefile,
version6.txt|8911 col 6| GvimExt/gvimext_ming.def, GvimExt/gvimext_ming.rc
version6.txt|9253 col 20| Problem:    Win32: gvim compiled with VC++ 7.0 run on Windows 95 does not show
version6.txt|9365 col 24| Problem:    Win32: The gvimext.dll build with Borland 5.5 requires another
version6.txt|9368 col 12| Files:	    GvimExt/Make_bc5.mak
version6.txt|10008 col 17| Files:	    nsis/gvim.nsi
version6.txt|10029 col 54| Problem:    Win32: The registry key for uninstalling GvimExt still uses "6.1".
version6.txt|10031 col 16| Files:	    src/GvimExt/GvimExt.reg
version6.txt|10615 col 12| src/bigvim.bat, src/feature.h, src/gui_beval.c, src/gui_beval.h,
version6.txt|11486 col 19| Problem:    Using gvim with Agide gives "connection lost" error messages.
version6.txt|12050 col 22| Problem:    Building gvimext.dll with Mingw doesn't work properly.
version6.txt|12053 col 16| Files:	    src/GvimExt/Make_ming.mak
version6.txt|12161 col 22| Problem:    Building gvimext.dll with Mingw doesn't work properly.
version6.txt|12163 col 16| Files:	    src/GvimExt/Make_ming.mak
version6.txt|12209 col 26| Problem:    Cannot build gvimext.dll with MingW on Linux.
version6.txt|12211 col 16| Files:	    src/GvimExt/Make_ming.mak
version6.txt|12231 col 26| Files:	    Makefile, src/gvim.exe.mnf, src/vim.rc
version6.txt|12315 col 24| Problem:    Win32: Run gvim, ":set go-=m", use Alt-Tab, keep Alt pressed while
version6.txt|12322 col 40| Problem:    Win32: The manifest causes gvim not to work. (Dave Roberts)
version6.txt|12324 col 16| Files:	    src/gvim.exe.mnf
version6.txt|13383 col 27| Files:	    Makefile, nsis/gvim.nsi
version6.txt|13392 col 51| Problem:    GUI: when changing 'cmdheight' in the gvimrc file the window
version6.txt|13405 col 35| Problem:    Win32: (un)installing gvimext.dll may fail if it was used.
version6.txt|13416 col 17| Files:	    nsis/gvim.nsi, src/dosinst.c, src/dosinst.h, src/uninstal.c
version6.txt|13546 col 17| Files:	    nsis/gvim.nsi, src/dosinst.c
version6.txt|13904 col 51| When using "set laststatus=2 cmdheight=2" in the .gvimrc you may only get one
version6.txt|14163 col 12| gives gvim focus and not moving the mouse after that, the effect
version7.txt|169 col 9| When a .gvimrc file exists then 'compatible' is off, just like when a ".vimrc"
version7.txt|1283 col 8| The $MYGVIMRC environment variable is set to the first found gvimrc file.
version7.txt|2006 col 62| from the terminal we actually get 3 lines for the cmdline in gvim.
version7.txt|2191 col 9| Motif: "gvim -iconic" opened the window anyway.  (David Harrison)
version7.txt|2482 col 29| GTK: Added a workaround for gvim crashing when used over an untrusted ssh
version7.txt|2520 col 2| "gvim -u NONE", ":set cursorcolumn", "C" in the second line didn't update
version7.txt|2552 col 8| Set $MYGVIMRC file to the first found .gvimrc file.
version7.txt|2686 col 18| Win32: Make the "gvim --help" window appear in the middle of the screen
version7.txt|2837 col 44| Win32: When a running Vim uses server name GVIM1 then "gvim --remote fname"
version7.txt|3112 col 17| Files:	    nsis/gvim.nsi
version7.txt|3234 col 45| Problem:    Win32: When compiled with SNIFF gvim may hang on exit.
version7.txt|3867 col 31| Problem:    GUI: when closing gvim is cancelled because there is a changed
version7.txt|3870 col 33| Solution:   Redraw when closing gvim is cancelled.
version7.txt|4130 col 27| Problem:    Win32: Using "gvim --remote-tab foo" when gvim is minimized while
version7.txt|4305 col 60| Problem:    Win32: Compiler warnings.  No need to generate gvim.exe.mnf.
version7.txt|4307 col 46| Files:	    src/Make_mvc.mak, src/eval.c, src/gvim.exe.mnf, src/misc2.c
version7.txt|4587 col 25| Problem:    Win32: When GvimExt is built with MSVC 2005 or later, the "Edit
version7.txt|4589 col 66| Solution:   Embed the linker manifest file into the resources of GvimExt.dll.
version7.txt|4591 col 16| Files:	    src/GvimExt/Makefile
version7.txt|5254 col 37| Problem:    Dropping a file name on gvim that contains a CSI byte doesn't work
version7.txt|5270 col 16| Files:	    src/GvimExt/gvimext.cpp
version7.txt|6489 col 16| Files:	    src/GvimExt/Makefile, src/dosinst.c, src/if_ole.cpp, src/if_ole.h,
version7.txt|6522 col 62| Problem:    Cscope still leaves behind temp files when using gvim.
version7.txt|6563 col 47| Problem:    Vimtutor only works with vim, not gvim.
version7.txt|6564 col 65| Solution:   Add the -g flag to vimtutor. (Dominique Pelle)  Add gvimtutor.
version7.txt|6565 col 30| Files:	    src/Makefile, src/gvimtutor, src/vimtutor, runtime/doc/vimtutor.1
version7.txt|6580 col 17| Problem:    src/gvimtutor is not distributed.
version7.txt|6751 col 26| Solution:   Only install gvimtutor when building a GUI version.
version7.txt|6946 col 17| Mac: On Leopard gvim, when using the mouse wheel nothing would happen until
version7.txt|6958 col 27| When files are dropped on gvim while the screen is being updated, ignore the
version7.txt|7070 col 31| Files:	    src/gui_w48.c, src/GvimExt/gvimext.cpp, src/Make_mvc.mak,
version7.txt|7234 col 5| The gvim executable is 32 bits, the installed gvimext.dll is either a 32 or 64
version7.txt|7237 col 20| The MS-Windows big gvim is now built with Python 2.7 and 3.1.2, Perl 5.12 and
version7.txt|7334 col 28| Support wide file names in gvimext. (Szabolcs Horvat)
version7.txt|8137 col 16| Problem:    In gvim "!grep a *.c" spews out a lot of text that can't be
version7.txt|8513 col 67| Problem:    Compilation problems after previous patch for Motif.  Gvim with
version7.txt|8651 col 34| Files:	    src/Make_mvc.mak, src/gvim.exe.mnf
version7.txt|8903 col 16| Files:	    src/GvimExt/gvimext.cpp
version7.txt|9296 col 66| Problem:    Wrong locale value breaks floating point numbers for gvim.
version7.txt|9554 col 25| Problem:    Can't build gvimext.dll on Win 7 x64 using MinGW (John Marriott)
version7.txt|9556 col 16| Files:	    src/GvimExt/gvimext.h
version7.txt|9597 col 29| Problem:    Cross-compiling GvimExt and xxd doesn't work.
version7.txt|9599 col 35| Files:	    src/INSTALLpc.txt, src/GvimExt/Make_ming.mak, src/Make_cyg.mak,
version7.txt|9625 col 61| Problem:    Misplaced assignment.  Duplicate build line for gvimext.dll.
version7.txt|10119 col 30| Win32: Put quotes around the gvim.exe path for the "Open with" menu entry.
version7.txt|10433 col 35| Files:	    src/Make_ming.mak, src/GvimExt/Make_ming.mak
version7.txt|10466 col 35| Files:	    src/Make_ming.mak, src/GvimExt/Make_ming.mak
version7.txt|10909 col 16| Files:	    src/GvimExt/Make_ming.mak, src/Make_ming.mak
version7.txt|10922 col 14| Problem:    "gvim -nb" is not interruptible.  Leaking file descriptor on
version7.txt|11663 col 34| Problem:    Warning for building GvimExt.
version7.txt|11665 col 16| Files:	    src/GvimExt/gvimext.def, src/GvimExt/gvimext_ming.def
version7.txt|11706 col 32| Problem:    Using fork() makes gvim crash on Mac when build with
version7.txt|11882 col 16| Files:	    src/GvimExt/gvimext.cpp
version7.txt|11926 col 16| Problem:    In gvim with iBus typing space in Insert mode doesn't work.
version7.txt|11980 col 13| Problem:    GvimExt sets $LANG in the wrong way.
version7.txt|11981 col 49| Solution:   Save the environment and use it for gvim. (Yasuhiro Matsumoto)
version7.txt|11982 col 16| Files:	    src/GvimExt/gvimext.cpp
version7.txt|11996 col 28| Problem:    With GTK, when gvim is full-screen and a tab is opened and using a
version7.txt|12262 col 16| Files:	    src/GvimExt/Make_cyg.mak, src/GvimExt/Make_ming.mak,
version7.txt|13374 col 13| Problem:    gvim starts up slow on Ubuntu 12.04.
version7.txt|13427 col 13| Problem:    gvim does not work when 'guioptions' includes "f". (Davido)
version7.txt|14050 col 53| src/xpm/include/xpm.h, src/Make_mvc.mak, src/bigvim.bat,
version7.txt|14051 col 12| src/bigvim64.bat, Filelist
version7.txt|14072 col 39| Problem:    MS-Windows: When starting gvim maximized 'lines' and 'columns' are
version7.txt|14526 col 18| Files:	    src/bigvim64.bat
version7.txt|14539 col 55| Problem:    Can't always find Win32.mak when building GvimExt.
version7.txt|14541 col 16| Files:	    src/GvimExt/Makefile
version7.txt|15496 col 18| Files:	    src/bigvim.bat
version7.txt|15664 col 10| src/GvimExt/gvimext.cpp, src/INSTALLvms.txt, src/Make_cyg.mak,
version7.txt|16041 col 36| Problem:    GTK vim not started as gvim doesn't set WM_CLASS property to a
version7.txt|17555 col 13| Problem:    gvim does not find the toolbar bitmap files in ~/vimfiles/bitmaps
version7.txt|17883 col 27| Files:	    Makefile, nsis/gvim.nsi
version7.txt|17900 col 17| Files:	    nsis/gvim.nsi
version8.txt|652 col 16| Files:	    src/GvimExt/Make_cyg.mak, src/Make_cyg.mak, src/xxd/Make_cyg.mak
version8.txt|2348 col 33| Problem:    Crash when starting gvim.  Issue 230.
version8.txt|3479 col 17| Files:	    nsis/gvim.nsi
version8.txt|4774 col 30| Solution:   Load the icon in GvimExt.
version8.txt|4775 col 17| Files:      src/GvimExt/gvimext.cpp, src/GvimExt/gvimext.h
version8.txt|4783 col 26| Problem:    Cannot build GvimExt.
version8.txt|4785 col 17| Files:      src/GvimExt/Makefile
version8.txt|4788 col 26| Problem:    Cannot build GvimExt with MingW.
version8.txt|4790 col 17| Files:      src/GvimExt/Make_ming.mak
version8.txt|5840 col 52| Files:      Filelist, runtime/vim.desktop, runtime/gvim.desktop, src/Makefile
version8.txt|6014 col 62| Problem:    test_utf8 fails on MS-Windows when executed with gvim.
version8.txt|6181 col 39| Problem:    Cannot run the tests with gvim.  Cannot run individual new tests.
version8.txt|6542 col 64| Problem:    On MS-Windows there is no target to run tests with gvim.
version8.txt|6543 col 25| Solution:   Add the testgvim target.
version8.txt|6566 col 19| Files:      src/bigvim.bat
version8.txt|6571 col 28| Files:      Makefile, nsis/gvim.nsi, nsis/gvim_version.nsh
version8.txt|6908 col 22| Problem:    Building GvimExt with VS2015 may fail.
version8.txt|6910 col 17| Files:      src/GvimExt/Makefile
version8.txt|6978 col 25| Problem:    Can't build GvimExt with SDK 7.1.
version8.txt|6980 col 35| Files:      src/Make_mvc.mak, src/GvimExt/Makefile
version8.txt|7008 col 17| Files:      src/GvimExt/Make_ming.mak, src/GvimExt/Make_cyg.mak,
version8.txt|7050 col 28| Files:      Makefile, nsis/gvim.nsi, src/os_win32.c, src/proto/os_win32.pro,
version8.txt|7141 col 44| Problem:    Test 92 and 93 fail when using gvim on a system with a non utf-8
version8.txt|7143 col 26| Solution:   Avoid using .gvimrc by adding -U NONE. (Yukihiro Nakadaira)
version8.txt|7280 col 26| Problem:    When running gvim in the foreground some icons are missing.
version8.txt|7910 col 17| Files:      src/GvimExt/Makefile, src/Make_mvc.mak, src/tee/Make_mvc.mak
version8.txt|8721 col 13| Problem:    GvimExt only loads the old libintl.dll.
version8.txt|8723 col 17| Files:      src/GvimExt/gvimext.cpp, src/GvimExt/gvimext.h
version8.txt|9441 col 25| Problem:    Can't build GvimExt with MingW.
version8.txt|9443 col 17| Files:      src/GvimExt/Make_ming.mak
version8.txt|10509 col 19| Problem:    Win32 gvim doesn't work with "dvorakj" input method.
version8.txt|10622 col 46| Problem:    Non-GUI specific settings in the gvimrc_example file.
version8.txt|10625 col 48| Files:      runtime/vimrc_example.vim, runtime/gvimrc_example.vim
version8.txt|11063 col 18| Files:      nsis/gvim.nsi
version8.txt|11941 col 13| gvim.exe which supports XPM using VC2015. (Ken Takata)
version8.txt|14000 col 38| Files:      Filelist, Makefile, nsis/gvim.nsi
version8.txt|14129 col 19| Files:      src/bigvim.bat, src/bigvim64.bat, src/Make_mvc.mak
version8.txt|14362 col 37| src/Make_mvc.mak, src/bigvim.bat
version8.txt|14372 col 19| Files:      src/bigvim.bat
version8.txt|14513 col 24| Problem:    MS-Windows gvim.exe does not have DirectX support.
version8.txt|14515 col 19| Files:      src/bigvim.bat
version8.txt|14815 col 39| runtime/doc/todo.txt, src/GvimExt/Makefile, src/Make_mvc.mak,
version8.txt|15491 col 33| Problem:    When using MSVC the GvimExt directory is cleaned twice.
version8.txt|15672 col 33| Problem:    Setting language in gvim on MS-Windows does not work when
version8.txt|17005 col 20| second gvim.  Don't use msg() when it would result in a dialog.
version8.txt|17085 col 28| Files:      Makefile, nsis/gvim.nsi
version8.txt|17098 col 49| Problem:    Menu test fails on MS-Windows using gvim.
version8.txt|18482 col 17| Files:      src/GvimExt/Makefile, src/Make_mvc.mak
version8.txt|18620 col 18| Files:      nsis/gvim.nsi
version8.txt|18662 col 18| Files:      nsis/gvim.nsi
version8.txt|19997 col 17| Files:      src/GvimExt/Makefile, src/Make_mvc.mak
version8.txt|20458 col 18| Files:      nsis/gvim.nsi
version8.txt|21061 col 18| Files:      nsis/gvim.nsi
version8.txt|21407 col 18| Files:      nsis/gvim.nsi
version8.txt|21737 col 54| Files:      src/msvc2015.bat, src/INSTALLpc.txt, src/GvimExt/Makefile,
version8.txt|21766 col 58| Files:      Filelist, Makefile, src/INSTALLpc.txt, src/bigvim.bat
version8.txt|21818 col 40| Solution:   Include both 32 and 64 bit GvimExt and related dll files. Remove
version8.txt|21820 col 35| Files:      nsis/README.txt, nsis/gvim.nsi, src/GvimExt/gvimext.cpp,
version8.txt|21853 col 28| Files:      Makefile, nsis/gvim.nsi
version8.txt|22087 col 28| Files:      Filelist, nsis/gvim.nsi, nsis/vimrc.ini, src/dosinst.c,
version8.txt|22424 col 32| src/Makefile, nsis/gvim.nsi
version8.txt|23039 col 37| src/Make_mvc.mak, src/bigvim.bat
version8.txt|23560 col 18| Files:      nsis/gvim.nsi
version8.txt|23937 col 44| Problem:    C syntax test fails when using gvim
version8.txt|23943 col 50| Problem:    C syntax test still fails when using gvim.
version8.txt|24383 col 47| Problem:    --clean loads user settings from .gvimrc.
version8.txt|24864 col 43| Problem:    Not easy to run one test with gvim instead of vim.
version8.txt|25416 col 13| Problem:    Gvim test gets stuck in dialog.
version8.txt|25583 col 18| Files:      nsis/gvim.nsi
vi_diff.txt|944 col 5| -U {gvimrc}	Vim: Read GUI initializations from {gvimrc} file.
workshop.txt|11 col 18| 3. Compiling vim/gvim for WorkShop			|workshop-compiling|
workshop.txt|12 col 16| 4. Configuring gvim for a WorkShop release tree		|workshop-configure|
workshop.txt|23 col 5| for gvim.  A workshop debug session will have a debugging window and an editor
workshop.txt|29 col 14| than the vim/gvim shipped with Visual WorkShop.  VWS users wishing to use vim
workshop.txt|43 col 18| 3. Compiling vim/gvim for WorkShop			*workshop-compiling*
workshop.txt|53 col 16| 4. Configuring gvim for a WorkShop release tree		*workshop-configure*
workshop.txt|55 col 71| There are several assumptions which must be met in order to compile a gvim for
workshop.txt|68 col 70| o We also suggest you use Motif for your gui.  This will provide gvim with
workshop.txt|80 col 31| symlink <VWS-install-dir>/bin/gvim to point to the vim in your newly installed
```

**Comment by k-takata** [comment]
CSApprox is not a part of the official Vim distribution, so it is out of scope.

**Comment by tonymec** [comment]
@k-takata : I know, it just happened to be included, and very noticeable, because it comes first in the `:helpgrep` output, which scans all `runtime/*.txt` files under the `'vimruntime'` directories

**Comment by brammool** [comment]
Tony wrote:

> Hm, it seems that Bram writes "gvim" but "GvimExt" almost everywhere,
> but that the author of the CSApprox plugin consistently uses "GVim" —
> see quickfix list below.
> 
> This said, with Vim there usually isn't "only one way" of doing
> things, there often are several key sequences that give the same
> result, and as long as the text is clear I wouldn't be paranoid about
> which letter to write in uppercase — except in code examples, where
> case consistency is usually required (a variable named fooBar is not
> the same as Foobar or FooBar or foobar or FOOBAR or, of course,
> foo_bar). In code, and e.g. in examples of command-lines to invoke the
> program, the Vim executable name must be in all lowercase, i.e. vim,
> view, vimdiff, gvim, gview, etc.

The executables are always lower case, that's clear.

Since Vim is a name, a product name, it starts with a capital.

In some cases VIM is used, but that should be an exception.

Now for GUI Vim, how to write it in normal text?  I would say that Vim is
the strongest part of the name, and prepending "g" is for the GUI
variant.  Thus gVim would be preferred.

If you think that every name should start with a capital, then you end
up with GVim.  But two capitals is a bit strange, I would only use that
at the start of a sentence.  Also, Gvim is strange, since it is dropping
the capital from Vim.  There is one exception: We are using GvimExt as a
directory name.  Changing that would be a hassle, so let's keep it.

I also think that gVim looks better.

-- 
hundred-and-one symptoms of being an internet addict:
205. You're constantly yelling at your spouse, family, roommate, whatever,
     for using the phone for stupid things...like talking.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
